pkg/antctl/raw/traceflow/command.go

```diff
@@ -112,46 +120,50 @@ func init() {
 	Command.Flags().BoolVarP(&option.nowait, "nowait", "", false, "if set, command returns without retrieving results")
 }
 
+func getk8sClient(cmd *cobra.Command) (kubernetes.Interface, antrea.Interface, error) {
+	kubeconfig, err := raw.ResolveKubeconfig(cmd)
+	if err != nil {
+		fmt.Println(err)
+		return nil, nil, err
+	}
+	k8sClientset, client, err := raw.SetupClients(kubeconfig)
+	if err != nil {
+		return nil, nil, fmt.Errorf("failed to create clientset: %w", err)
+	}
+	return k8sClientset, client, nil
+}
+
 func runE(cmd *cobra.Command, _ []string) error {
 	option.timeout, _ = cmd.Flags().GetDuration("timeout")
 	if option.timeout > time.Hour*12 {
-		fmt.Println("Timeout cannot be longer than 12 hours")
+		cmd.OutOrStdout().Write([]byte("Timeout cannot be longer than 12 hours"))
```

Comment:
You may use the format `fmt.Fprintf(cmd.OutOrStdout(), Timeout cannot be longer than 12 hours)` to print out the message.